test/integration/expression-tests/image/coalesce/test.json

```diff
@@ -4,7 +4,11 @@
     "type": "resolvedImage"
   },
   "inputs": [
-    [{}, {}]
+    [
+        {},
```

Comment:
First object of an input array represents evaluation context (zoom, heatmap), let's add new key `availableImages` with array of strings.
```
    [
        { 'availableImages': ["monument-15"] },
        {},
    ]
```